components/menu/index.css

```
@@ -370,6 +370,8 @@
 	position: relative;
 	align-items: center;
 
+	border-radius: var(--mod-menu-item-corner-radius, var(--spectrum-menu-item-corner-radius));
```

```suggestion
	border-radius: var(--spectrum-menu-item-corner-radius);
```

We avoid adding new `--mod`s unless they're specifically requested by product or indicated by design.